hadoop-hdfs-project/hadoop-hdfs-rbf/pom.xml

```diff
@@ -251,10 +248,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
               <goal>run</goal>
             </goals>
             <configuration>
-              <tasks>
+              <task>
                 <copy file="src/main/resources/hdfs-rbf-default.xml" todir="src/site/resources"/>
                 <copy file="src/main/xsl/configuration.xsl" todir="src/site/resources"/>
-              </tasks>
+              </task>
```

Comment:
Was supposed to use target, right?
From the description
```
Use target instead of tasks.
```
Other places you moved tasks -> target only. Any specific reason here?

Reply:
My bad. You are right. I fixed it.